docs/install/windows_setup.md

```diff
@@ -114,81 +114,138 @@ For CPU-only package:
   install.packages("mxnet")
 ```
 
-For GPU-enabled package:
+#### Building MXNet from Source Code(CPU)
+1. Clone the MXNet github repo.
 
-```r
-  cran <- getOption("repos")
-  cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU"
-  options(repos = cran)
-  install.packages("mxnet")
+```sh
+git clone --recursive https://github.com/dmlc/mxnet
```

Comment:
"https://github.com/dmlc/mxnet" is out of date. Please use "https://github.com/apache/incubator-mxnet"